mlir/test/Conversion/TosaToLinalg/tosa-to-linalg-pipeline.mlir

```diff
@@ -1,16 +1,6 @@
 // RUN: mlir-opt %s --split-input-file --tosa-to-linalg-pipeline -verify-diagnostics
 
 
-// -----
-
-// check that -tosa-validate of stateful ops kick in
-func.func @test_variable_write_shape(%arg0: tensor<1x4x8xi8>) -> () {
```

Comment:
It seems that a useful test case remains for `--tosa-to-linalg-pipeline` here. I’m wondering why it was removed? Could we update it as shown below instead?

Reply:
That is a useful test; however, this shape check has been moved to the built-in operation verifier, so it can no longer be triggered during validation. Actually, it is noew covered in the verifier.mlir. Sound right to you?
Additionally, since the --tosa-to-linalg-pipeline option can opt-out TOSA validation during the passes, for this test file, it's unclear why we need to confirm that validation is invoked during the TOSA-to-Linalg conversion.

Reply:
Understood now, thanks Tatwai.